03-lessons_from_user_designers.Rmd

```diff
@@ -0,0 +1,141 @@
+
+## General principles about user-friendliness  
+
+While finding out about your user community is critical, there's also principles we can discuss that are common to all users/humans.
+
+#### Humans like visuals
+
+Humans are drawn to intuitive visuals.
+Visuals are efficient means of communication and help users absorb information better than long-winded paragraphs (though visuals need an accompanying explanation too).
+
+Sometimes this is particularly helpful for complicated concepts. 
+For example, BEDtools (@Quinlan2010) allows for the manipulation of genomic sequences in BED files. 
+Some of these principles can be complicated to visualize, but the authors of BEDtools do a great job of using visuals to explain each function: 
+
+![](https://docs.google.com/presentation/d/1cd434bkLer_CJ04GzpsZwzeEA9gjc5Ho6QimiHPbyEg/export/png?id=1cd434bkLer_CJ04GzpsZwzeEA9gjc5Ho6QimiHPbyEg&pageid=gcd0e3791ab_0_44)
+
+#### Stay away from jargon
+
+What someone considers jargon is a very relative to their own experiences.  
+Terms that may seem like common knowledge to you may be foreign to your user. 
+
+For example, something seemingly commonplace to you like `TSV` may not be something a user does not understands what it means. 
+To help smooth over jargon-related barriers, spell out abbreviations the first time you mention them (e.g. 'tab separated values').
+This doesn't mean you have to have long winded explanations of every term, instead you can quickly link out to an article or website with information about a term you've used (e.g. [tab separated values file (TSV)](https://en.wikipedia.org/wiki/Tab-separated_values)). 
+This has the benefit of saving you and your user's time without making your explanations too long winded. 
+
+#### Navigability: Make help easy to find
+
+When your user has a question in mind, it can make their experience a lot more stress-free if they can find their answer with ease.
+Its not always obvious to a user where you have put an answer to something, particularly if user is already frustrated; deep in the troubleshooting trenches!
+For these reasons, navigability is important for your tool and documentation.
+
+Increasing your tools' navigability:     
+- Workshop an [intuitive layout](https://www.oreilly.com/library/view/designing-interfaces/0596008031/ch04.html).    
+- Minimize the number of clicks for finding help! -- The more important a piece of information is to a users' ability to use your tool, the more it should be in the forefront of your tool!  
+- Be consistent about where to find things. Once a user has caught on to your pattern of how you store things in your tool, don't switch it -- patterns are super helpful!  
+
+Articles about user-friendly principles:   
+
+- [Principles of User Interface Design](http://bokardo.com/principles-of-user-interface-design/) by @Porter.   
+- [Golden Rules of UI Design](https://xd.adobe.com/ideas/process/ui-design/4-golden-rules-ui-design/) by @Babich.  
+
```

Comment:
I would like to put an example image of a bioinformatics tool with a great layout here. Haven't found a great candidate for this yet though.